modules/@apostrophecms/piece-type/ui/apos/components/AposPiecesManagerDisplay.vue

```diff
@@ -48,18 +48,21 @@
           />
         </td>
         <td
-          class="apos-table__cell" v-for="header in headers"
+          class="apos-table__cell apos-table__cell--pointer" v-for="header in headers"
           :key="item[header.name]"
+          @click="$emit('open',item._id)"
         >
           <a
-            v-if="header.name === 'url'" class="apos-table__link"
+            v-if="header.name === '_url' && item[header.name]"
+            class="apos-table__link"
             :href="item[header.name]"
+            @click.stop
           >
-            <LinkIcon :size="12" />
+            <link-icon :size="14" />
           </a>
           <button
             v-else-if="header.name === 'title'"
-            @click="$emit('open',item._id)"
```

Comment:
I'm leaving this button click event here because it's going to do the easy work in keyboard control situations rather than fuss with `tabindex` on the table cell when one also has a link in it.